Makefile

```diff
@@ -5,6 +5,7 @@ BUILDTAGS=seccomp
 export GOPATH:=$(CURDIR)/Godeps/_workspace:$(GOPATH)
 
 all:
+	ln -sfn $(CURDIR) $(CURDIR)/Godeps/_workspace/src/github.com/opencontainers/runc
 	go build -tags "$(BUILDTAGS)" -o runc .
```

Comment:
The test case is surely fixable, but I have the same question as @tianon 

> I'm curious how the godeps tool handles extra content like this inside the `_workspace` directory -- does it consider that directory to be its personal space and just blast away everything inside? Does it maybe have an option for creating/keeping this symlink automatically for us?

I think the better option is to have some change in `Godeps/Godeps.json` and let go tools create this symlink automatically for us, but if go tools don't like symlink, I doubt there is a such solution.